config/secrets.yml.sample

```diff
@@ -1,17 +0,0 @@
-# Be sure to restart your server when you modify this file.
```

Comment:
Leave this file it will sample for those people that will maintain this project later.